packages/runtime/src/account.ts

```diff
@@ -492,3 +492,185 @@ export class BaseModelI implements BaseModel {
     throw new Error("get_obj_for_encoding Not Implemented");
   }
 }
+
+//list of 16 predefined accounts 
+export const defaultSDKAccounts = {
```

Comment:
let's put this somewhere in `constants` (or maybe `default_accounts.ts`)

Reply:
I don't think we should put all possible constants to `contstants`. I think here is OK, it's `accounts` module, so easy to find. `default_accounts.ts` is also OK.